spec/seeing_is_believing_spec.rb

```diff
@@ -523,7 +523,7 @@ def meth
     end
   end
 
-  it 'does not kill parent processes' do
+  it 'does not kill parent processes', :unless => RSpec::Support::OS.windows? do
```

Comment:
I'm a little surprised the test [above this](https://github.com/avdi/seeing_is_believing/blob/1c36225b9fc767d98dd7974e00af54133c6b0ba1/spec/seeing_is_believing_spec.rb#L513) works. I guess it just explodes in an uninteresting way on Windows. Makes me uncomfortable with test :/